src/inbox.rs

```diff
@@ -89,7 +89,7 @@ impl<A> ChanInner<A> {
         // If len < cap after popping this message, try fulfill at most one waiting sender
         if capacity.map_or(false, |cap| cap == self.ordered_queue.len()) {
             match self.try_fulfill_sender(MessageType::Ordered) {
-                Some(SentMessage::ToOneActor(msg)) => self.ordered_queue.push_back(msg.val),
+                Some(MessageKind::ToOneActor(msg)) => self.ordered_queue.push_back(msg.val),
```

Comment:
Same with this rename!

Reply:
This was done since SentMessage became a struct with more stuff inside of it